st2client/st2client/commands/action.py

```diff
@@ -424,6 +456,17 @@ def _format_error_result(self, result):
                 result['traceback'])
         return result
 
+    def _get_inherited_env_vars(self):
```

Comment:
@lakshmi-kannan 

> lakstorm [03:18]
So it appears we make a copy of the env variables when the CLI is executed. This is then available to actions.
lakstorm [03:18]
So confused why we did this.
lakstorm [03:19]
so even though auth_token is emtpy, in 0.12, the env variables are passed down from CLI. (edited)
lakstorm [03:20]
I am really confused about this whole thing.

That was a feature request by @jtopjian. We only do that if `--inherit-env` flag is provided.